healthcare/v1beta1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceListHistory.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package snippets.healthcare.fhir.resources;
+
+// [START healthcare_list_resource_history]
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.healthcare.v1beta1.CloudHealthcare;
+import com.google.api.services.healthcare.v1beta1.CloudHealthcareScopes;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.client.methods.RequestBuilder;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.impl.client.HttpClients;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Collections;
+
+public class FhirResourceListHistory {
+  private static final String FHIR_NAME = "projects/%s/locations/%s/datasets/%s/fhirStores/%s";
+  private static final JsonFactory JSON_FACTORY = new JacksonFactory();
+  private static final NetHttpTransport HTTP_TRANSPORT = new NetHttpTransport();
+
+  public static void fhirResourceListHistory(String resourceName)
+      throws IOException, URISyntaxException {
+    // String resourceName =
+    //    String.format(
+    //        FHIR_NAME, "project-id", "region-id", "dataset-id", "store-id", "fhir-id");
+
+    // Initialize the client, which will be used to interact with the service.
+    CloudHealthcare client = createClient();
+
+    HttpClient httpClient = HttpClients.createDefault();
+    String uri = String.format(
+        "%sv1beta1/%s/_history", client.getRootUrl(), resourceName);
```

Comment:
Should the v1beta1 be defined near the top in all caps so users could easily change it?

Reply:
Are users meant to copy this code directly? I always advocated for making this code as copy-pastable as possible, but it seemed like the sample format didn't go in that direction.